src/components/ha-control-circular-slider.ts

```diff
@@ -171,6 +173,18 @@ export class HaControlCircularSlider extends LitElement {
         this._localHigh = this.high;
       }
     }
+
+    if (
+      (changedProps.has("_localValue") &&
+        changedProps.get("_localValue") == null) ||
+      (changedProps.has("_localLow") &&
+        changedProps.get("_localLow") == null) ||
+      (changedProps.has("_localHigh") &&
+        changedProps.get("_localHigh") == null) ||
```

Comment:
Why do you do these checks? Is it to catch the first update?

Reply:
The target dot is removed from the dom if the values are undefined so we need to recreate hammerjs.